ci/scripts/run_ci.sh

```diff
@@ -43,29 +43,46 @@ else
   exit 1
 fi
 
-pr_list_file="open_pr_list"
+pr_list_dbfile="${GFS_CI_ROOT}/open_pr_list.db"
 
-if [[ -s "${GFS_CI_ROOT}/${pr_list_file}" ]]; then
-  pr_list=$(cat "${GFS_CI_ROOT}/${pr_list_file}")
-else
-  echo "no PRs to process .. exit"
+#NOTE: last pipe with head -2 limits no more tha two PRs at a time
+pr_list=""
+if [[ -f "${pr_list_dbfile}" ]]; then
+  pr_list=$("${HOMEgfs}/ci/scripts/pr_list_database.py" --display "${pr_list_dbfile}" | grep -v Failed | grep Open | grep Built | awk '{print $1}' | head -2) || true
```

Comment:
Why not use an SQL call that only selects certain statuses, instead of doing all these greps?

Reply:
Also, the number of concurrent cases should be configurable somewhere instead of hard-coded.

Reply:
OK, I didn't want the interface to be to complicated.  Lets do this work on the interfaces as part of your suggested updates on the positional arguments.  I will make the code changes for the configuration of the throttle values for this PR though, ok?